### ISSUE TITLE
Update tests and patches to support Drupal 8.4

### ISSUE THREAD

**Issue by dearmond** [ucdavis/sitefarm_seed#171]

**Comment by dearmond** [comment]
These tests are still failing:
```
tests/features/article.feature:50
    tests/features/article.feature:97
    tests/features/ckeditor.feature:306
    tests/features/ckeditor.feature:359
    tests/features/event.feature:88
    tests/features/event.feature:112
    tests/features/external_media_embed.feature:11
    tests/features/page.feature:40
    tests/features/person.feature:87
    tests/features/photo_gallery.feature:25
    tests/features/photo_gallery.feature:53
    tests/features/rss_feed.feature:31
    tests/features/xml_sitemap.feature:17
    tests/features/xml_sitemap.feature:30
```

**Comment by devnich** [comment]
FYI, I'm still getting an error when trying to apply the patch against 8.4.4. The patch version it's trying to apply is:
"Profile Inheritance": "https://www.drupal.org/files/issues/1356276-408--8.4.x.patch"

**Comment by eguerin-ucsf** [comment]
There is a specific patch for 8.4.4, you'll need to apply that in your parent Composer file.
https://www.drupal.org/files/issues/1356276-419--8.4.4.patch

**Comment by eguerin-ucsf** [comment]
Looks like the last patch is also failing as the sub-profile will not install.  See ucdavis#183 and ucdavis#184

**Comment by mrkmiller** [comment]
Fixed by #198
